src/fragments/ui/auth/flutter/authenticator.mdx

```diff
@@ -0,0 +1,335 @@
+## Basic Integration
+
+The Amplify-Flutter Authenticator Widget is a Material-compatible sign-up and sign-in experience which reacts to your Amplify configuration. It automatically detects the authentication state and updates the UI. If the user is signed in, the underlying component (typically your app's home widget) is displayed; otherwise, signin/signup controls are displayed.
+
+To begin, follow the steps for configuring the Amplify Auth category from the [getting started walkthrough](/lib/auth/getting-started). Be sure to also add the Amplify-Flutter Authenticator to your `pubspec.yaml` file:
+
+```yaml
+dependencies:
+  amplify_authenticator: ^1.0.0-rc.0
+```
+
+You will also need to import the Amplify-Flutter Authenticator into your app:
+
+```dart
+import 'package:amplify_authenticator/amplify_authenticator.dart';
+```
+
+Then, update your main app widget's `build` method:
+
+```dart
+@override
+	Widget build(BuildContext context) {
+	  return MaterialApp(
+	    title: 'Authenticator Demo',
+	    // The Authenticator component should be a descendant of your MaterialApp.
+	    home: Authenticator(
+	      child: const Scaffold(
+	        body: Center(
+	          child: Text('You are logged in!'),
+	        ),
+	      ),
+	    ),
+	  );
+	}
+```
+
+If you run your application, you should now see the Amplify-Flutter Authenticator Widget!
+
+
+![Basic SignIn](/images/flutterAuthenticator/flutter_authenticator_signin_base.png)
+
+## Customizable Forms
+
+Typically, the Amplify-Flutter Authenticator will use your Amplify configuration to determine which inputs are displayed in a form. It also offers the ability to override this default behavior and customize three of its UI forms:
+
+- SignInForm
+- SignUpForm
+- ConfirmSignInNewPasswordForm
+
+By leveraging this customization, you can control the input fields that are displayed in the form, the validation that is run against the field upon submission and whether or not the field is considered optional.
+
+For example, let's say that you would like to allow (but not force) your users to provide their address. You also want to collect a custom Cognito attribute that you created in your User Pool after your config was generated. Finally, you want to ensure that all usernames include the term 'amplify'.
+
+You can satisfy all three of these requirements by passing a custom signup form:
+
+```dart
+String? _validateUsername(UsernameInput? input) {
+  final username = input?.username;
+  if (username == null || username.isEmpty) {
+    return 'Username cannot be empty';
+  }
+  bool containsAmplify = username.contains('amplify');
+  if (!containsAmplify) {
+    return 'Username needs to include amplify';
+  }
+  return null;
+}
+
+final authenticator = Authenticator(
```

Comment:
Please reformat this section - there is a mix of 4 spaces and 2 spaces for indentation. It should always be 2 for Dart.